### PR TITLE
Import for `WeaviateVectorStore` from langchain_weaviate in `_get_builtin_translator`

### PR DESCRIPTION
**Pull request by keenborder786**
- **Description:** The `WeaviateVectorStore` from `langchain_weaviate` was missing in `_get_builtin_translator` which have been added in this PR.
  - **Issue:** #28579

### CODE REVIEW

File: libs/langchain/langchain/retrievers/self_query/base.py [hunk 1]
```diff
@@ -94,6 +94,7 @@ def _get_builtin_translator(vectorstore: VectorStore) -> Visitor:
     from langchain_community.vectorstores import (
         Pinecone as CommunityPinecone,
     )
+    from langchain_weaviate.vectorstores import WeaviateVectorStore
```
Comment (baskaryan) [inline libs/langchain/langchain/retrievers/self_query/base.py:97]: can we conditionally import the way we do with elasticsearch in L141

Comment (keenborder786) [reply libs/langchain/langchain/retrievers/self_query/base.py:97]: Sure

Comment (keenborder786) [reply libs/langchain/langchain/retrievers/self_query/base.py]: Done

**Comment by efriis** [comment]
I think this may have been handled in a different PR simultaneously sorry about that! The remaining change seems unrelated. Feel free to reopen with a diff title if important

**Pull request closed without merging**